nixnet/database/_find_object.py

```
@@ -26,28 +26,31 @@ def find_object(
     from nixnet.database._signal import Signal
     from nixnet.database._subframe import SubFrame
 
-    found_object = None  # type: _database_object_base.DatabaseObjectBase
-    found_handle = _funcs.nxdb_find_object(parent_handle, object_class, object_name)
+    if isinstance(object_class, Cluster):
+        class_enum = constants.ObjectClass.CLUSTER
+    elif isinstance(object_class, Ecu):
+        class_enum = constants.ObjectClass.ECU
+    elif isinstance(object_class, Frame):
+        class_enum = constants.ObjectClass.FRAME
+    elif isinstance(object_class, LinSched):
+        class_enum = constants.ObjectClass.LIN_SCHED
+    elif isinstance(object_class, LinSchedEntry):
+        class_enum = constants.ObjectClass.LIN_SCHED_ENTRY
+    elif isinstance(object_class, Pdu):
+        class_enum = constants.ObjectClass.PDU
+    elif isinstance(object_class, Signal):
+        class_enum = constants.ObjectClass.PDU
+    elif isinstance(object_class, Signal):
+        class_enum = constants.ObjectClass.SIGNAL
+    elif isinstance(object_class, SubFrame):
+        class_enum = constants.ObjectClass.SUBFRAME
+    else:
+        # arbitrary value for class_enum so mypy sees a value in each condition
+        class_enum = constants.ObjectClass.CLUSTER
```

I'd actually recommend a `raise ValueError("Unsupported database object")`

---

Oh, missed the raise below :)